cmd/awscognitosource/Makefile

```diff
@@ -1,6 +1,5 @@
 PACKAGE = awscognitosource
 PACKAGE_DESC = Triggermesh AWS Cognito Event Source
 
-GOPKGS = ../../pkg/...
-
 include ../../scripts/inc.Makefile
+include ../../scripts/inc.Target
```

Comment:
Looks like your editor removes the final new line from some files, but not from others. In general I'd prefer to keep them (most editors do by default), because files without a final new line cause weird things in shell prompts when you `cat` them:
```console
acotten:/git/aws-event-sources $ cat cmd/awscognitosource/Makefile
PACKAGE = awscognitosource
PACKAGE_DESC = Triggermesh AWS Cognito Event Source

GOPKGS = ../../pkg/...

include ../../scripts/inc.Makefile
include ../../scripts/inc.Targetacotten:/git/aws-event-sources $
```

Reply:
Yea. I recently reset my editor. Need to configure it to add the new line